addon/helpers/ember-sparkles/axis.js

```diff
@@ -10,8 +10,13 @@ export function emberSparklesAxis([ scale ], { position, tickFormat, ticks, widt
 
   let result = axisFn().scale(scale);
 
-  if (tickFormat) {
-    result.tickFormat(timeFormat(tickFormat));
+  switch(typeof tickFormat) {
```

Comment:
It looks like we might be all looking at this from different angles. I propose to pull this logic out of this helper and have the context decide what tickFormat should be applied. We can create something like `d3-tick-format` helper that'll accept a format and return a function that this helper will use. 

This will allow the context to define what tickFormat should be used and this helper will not need to consider different cases. The helper that we create can have a fallback for anything other than a string or a function.

What do you think?